### DISCUSSION TITLE
Annotations in Android 11.0.0-pre0

### DISCUSSION THREAD

**Discussion by mtpdog** [maplibre/maplibre-native#2141] [General]
**Describe the bug**
What is the preferred way to work with annotations in 11.0.0-pre0?
- It looks like 'org.maplibre.gl:android-plugin-annotation-v9:2.0.2' had become mostly incompatible because of the renaming from Mapbox to MapLibre.
- new annotation classes described [here](https://github.com/maplibre/maplibre-native/blob/main/design-proposals/2023-06-17-android-annotations.md) cannot be found.
- [CustomDrawableLayer](https://github.com/maplibre/maplibre-native/pull/1908) is also absent.

Could you give some examples of how to add for example circle with text and some custom drawable image to the map?

**Expected behavior**
Sample project with all major types of annotations added to the map.

**Platform information (please complete the following information):**
 - OS: Android
 - Platform Android
 - Version 11.0.0-pre0

**Comment by louwers** [comment]
Something went wrong while publishing the pre-release to Maven Central, so I will have to make another release.

After the pre-release is published to Maven Central, we can update the Annotations plugin which is kept in a separate repository. https://github.com/maplibre/maplibre-plugins-android

This is probably a good idea since the Annotations API is still a WIP, the tracking issue is here: https://github.com/maplibre/maplibre-native/issues/1491 (cc @fynngodau)

The custom drawable layer is only for iOS right now (you can only use it with C++ directly), but the existing custom layer based on Open GL ES continues to work.

**Comment by mtpdog** [comment]
What is the correct dependency to implement 11.0.0-pre0? 
This is not found on Maven Central:
api ('org.maplibre.gl:android-sdk:11.0.0-pre0') 

I could only install it using local aar file
